tinyagentos/routes/desktop_browser/suggest.py

```
@@ -0,0 +1,70 @@
+
+from tinyagentos.auth import get_current_user
+from tinyagentos.routes.desktop_browser import router
+
+
+@router.get("/api/desktop/browser/suggest")
+async def suggest(
+    profile_id: str,
+    q: str,
+    request: Request,
+    limit: int = 8,
+    current_user: dict[str, Any] = Depends(get_current_user),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)